raster-foundry/raster-foundry
# [WIP] Configure Ingest jobs to use their own batch queue

## Overview

Add a configuration option for a separate `ingestJobQueue`, and modify `kickoffSceneIngest` to submit jobs to `ingestJobQueue`.

### Checklist

~- [ ] Styleguide updated, if necessary~
~- [ ] Swagger specification updated, if necessary~
~- [ ] Symlinks from new migrations present or corrected for any new migrations~

### Demo

Optional. Screenshots, `curl` examples, etc.

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.


## Testing Instructions
 * Run `scripts/bootstrap` to pull down the new `.env` file
 * In `.env`, set `ENVIRONMENT=staging`
 * Start up the local development app with `scripts/server`
 * Add a scene to one of your projects.
 * On the AWS Batch console, make sure your job gets routed to `queueStagingIngest`. The job should fail since the project doesn't exist on staging.

Needs azavea/raster-foundry-deployment#96
Closes azavea/raster-foundry-platform#162
